File: atcoder/abc146/abc146_a.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.HashMap;

class Main {

	public static void main(String[] args) throws Exception {
		
		InputStreamReader is= new InputStreamReader(System.in);
		BufferedReader br = new BufferedReader(is);
			
		String day = br.readLine();
		
		HashMap<String, Integer> map = new HashMap<String, Integer>();
		map.put("MON", 6);
		map.put("TUE", 5);
		map.put("WED", 4);
		map.put("THU", 3);
		map.put("FRI", 2);
		map.put("SAT", 1);
		map.put("SUN", 7);
		
		System.out.println(map.get(day));
			
	}
	
}

File: atcoder/abc146/abc146_a.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main() {
    1
    val input = BufferedReader( InputStreamReader( System.`in` ) )
    val dayIn = input.readLine()

    val hashMap = HashMap<String, Int>();
    //SUN, MON, TUE, WED, THU, FRI, or SAT
    hashMap.put( "SUN", 7 );
    hashMap.put( "MON", 6 );
    hashMap.put( "TUE", 5 );
    hashMap.put( "WED", 4 );
    hashMap.put( "THU", 3 );
    hashMap.put( "FRI", 2 );
    hashMap.put( "SAT", 1 );

    print( hashMap.get( dayIn ) )

}